src/index.js

```diff
@@ -1,12 +1,16 @@
-import QS from 'qs';
-import Cookie from 'js-cookie';
-import EventEmitter from 'events';
-import { debounce, isFunction, assign, omitBy, isNil, uuid } from './utils';
+import QS from "qs";
```

Comment:
hm, how come strings are changing to be double-quoted? elsewhere, we always use single-quotes.

Reply:
all of these cosmetic changes were done because of prettify lol. I can revert if you want

Reply:
:) yes, if you don't mind, think that would be preferable. we can bring prettier into wt-client at another time